docs/index.md

```diff
@@ -0,0 +1,44 @@
+# django-audittrail
+
+This project allow you to register and store user actions in database and then do whatever you want with this data.
+You can assing any model entity you created in your application to `content_object` which uses `GenericForeignKey` field. In addition it contains `user_id` field which is not related
+to the basic `User` model so you can assign id of user, regardless `User` instance you use.
+
+## Requirements
+  
+* python 3.6+
+* Django 2.0+
+
+django-audittrail may work as well on previous versions of django and python but it has not be tested.
```

Comment:
"django-audittrail may as well work on previous versions of Django and Python, however it has not yet been tested."